x-pack/plugins/elastic_assistant/server/lib/langchain/tools/open_alerts/get_open_alerts_query.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const getOpenAlertsQuery = ({
+  alertsIndexPattern,
+  size,
+}: {
+  alertsIndexPattern: string;
+  size: number;
+}) => ({
+  allow_no_indices: true,
+  body: {
+    fields: [
+      {
+        field: '*',
+        include_unmapped: true,
+      },
```

Comment:
nit: as we discussed this could be filtered to only return configured `allow` field.